### DISCUSSION TITLE
Notify Push: No such file or directory

### DISCUSSION THREAD

**Discussion by DrPleb** [nextcloud/all-in-one#3412] [Questions] [answered]
### Steps to reproduce
1. Start master container and all other containers
2. Wait that all containers to start
3. All containers start but notify push not

### Expected behavior 
Notify push container must start

### Actual behavior 
Notify push container won't start. In the log it says:
/start.sh: line 36: /nextcloud/custom_apps/notify_push/bin/x86_64/notify_push: No such file or directory

### Host OS 
Ubuntu 22.04.3 LTS

#### Nextcloud AIO version 
7.2.1

#### Current channel 
Stable latest

#### Other valuable info 
I installed Nextcloud Aio, at the start it says that there was an update of mastercontainer before domain setup, i did it, then setup domain, all correctly but the notify push container won't start. I also noticed that in the app store I don't have any app to search, I can only see the app already installed

![image](https://github.com/nextcloud/all-in-one/assets/132892195/74e307b1-b6b0-4221-b633-fbe4c744e2f5)
![image](https://github.com/nextcloud/all-in-one/assets/132892195/dd0cde4e-9e5d-4739-8e25-e683d1872b82)
![image](https://github.com/nextcloud/all-in-one/assets/132892195/a9bb90ca-e755-48ba-afd5-dce71445d22d)

**Comment by szaimen** [comment]
Hi, can you post the nextcloud container logs here?

**Comment by DrPleb** [comment]
> Hi, can you post the nextcloud container logs here?
       
------------------------------
 2023-09-21 11:37:59.26249+02
(1 row)

+ '[' -f /dev-dri-group-was-added ']'
++ find /dev -maxdepth 1 -mindepth 1 -name dri
+ '[' -n '' ']'
+ set +x
Installing imagemagick via apk...
Enabling Imagick...
Configuring Redis as session handler...
Setting php max children...
Applying one-click-instance settings...
System config value one-click-instance set to boolean true
System config value one-click-instance.user-limit set to integer 100
System config value one-click-instance.link set to string https://nextcloud.com/all-in-one/
support already enabled
Adjusting log files...
System config value upgrade.cli-upgrade-link set to string https://github.com/nextcloud/all-in-one/discussions/2726
System config value logfile set to string /var/www/html/data/nextcloud.log
Config value logfile for app admin_audit set to /var/www/html/data/audit.log
System config value updatedirectory set to string /nc-updater
Applying network settings...
System config value davstorage.request_timeout set to integer 3600
System config value trusted_domains => 1 set to string mydomain.com
System config value overwrite.cli.url set to string https://mydomain.com/
System config value htaccess.RewriteBase set to string /
.htaccess has been updated
System config value dbpersistent set to boolean true
System config value files_external_allow_create_new_local set to boolean false
Error: Could not download app notify_push
System config value trusted_proxies => 0 set to string 127.0.0.1
System config value trusted_proxies => 1 set to string ::1
Config value base_endpoint for app notify_push set to https://mydomain.com/push
Error: Could not download app richdocuments
Config value wopi_url for app richdocuments set to https://mydomain.com/
System config value allow_local_remote_servers set to boolean true
No ipv6-address found for mydomain.com.
Config value wopi_allowlist for app richdocuments set to hidden_ip,127.0.0.1/8,192.168.0.0/16,172.16.0.0/12,10.0.0.0/8,fd00::/8,::1
Error: Could not download app spreed

                                                               
  There are no commands defined in the "talk:turn" namespace.  
                                                               


                                                               
  There are no commands defined in the "talk:turn" namespace.  
                                                               


                                                               
  There are no commands defined in the "talk:stun" namespace.  
                                                               


                                                               
  There are no commands defined in the "talk:stun" namespace.  
                                                               


                                                               
  There are no commands defined in the "talk:stun" namespace.  
                                                               


                                                                    
  There are no commands defined in the "talk:signaling" namespace.  
                                                                    


                                                                    
  There are no commands defined in the "talk:signaling" namespace.  
                                                                    

Error: Could not download app files_antivirus
Config value av_mode for app files_antivirus set to daemon
Config value av_port for app files_antivirus set to 3310
Config value av_host for app files_antivirus set to nextcloud-aio-clamav
Config value av_stream_max_length for app files_antivirus set to 104857600
Config value av_max_file_size for app files_antivirus set to 104857600
Config value av_infected_action for app files_antivirus set to only_log
System config value enabledPreviewProviders => 0 set to string OC\Preview\Imaginary
System config value preview_imaginary_url set to string http://nextcloud-aio-imaginary:9000
Error: Could not download app fulltextsearch
Error: Could not download app fulltextsearch_elasticsearch
Error: Could not download app files_fulltextsearch

                                                                    
  There are no commands defined in the "fulltextsearch" namespace.  
                                                                    


                                                                               
  There are no commands defined in the "fulltextsearch_elasticsearch" namespa  
  ce.                                                                          
                                                                               


                                                                          
  There are no commands defined in the "files_fulltextsearch" namespace.  
                                                                          

Waiting 10s before activating FTS...
Activating fulltextsearch...

                                                                    
  There are no commands defined in the "fulltextsearch" namespace.  
                                                                    

Fulltextsearch failed. Could not index.
Feel free to follow https://github.com/nextcloud/all-in-one/discussions/1709 if you want to skip the indexing in the future.
[21-Sep-2023 11:38:42] NOTICE: fpm is running, pid 456
[21-Sep-2023 11:38:42] NOTICE: ready to handle connections
Activating Collabora config...

                                                                   
  There are no commands defined in the "richdocuments" namespace.

**Comment by szaimen** [comment, marked as answer]
> Error: Could not download app files_antivirus

all of these are your issues.

Can you check if enabling ipv6 correctly makes it work? 
See https://github.com/nextcloud/all-in-one/blob/main/docker-ipv6-support.md

**Reply by DrPleb** [reply]
Hi, I cleaned my istance following the guide on the nextcloud-aio page. I edited the docker daemon file as the link you posted says, but now I don't understand how to create the new istance with ipv6 enabled. Usually this is the command for the new istance that i launch:

sudo docker run \
--sig-proxy=false \
--name nextcloud-aio-mastercontainer \
--restart always \
--publish 8080:8080 \
--env APACHE_PORT=1001 \
--env TALK_PORT=3480 \
--env APACHE_IP_BINDING=0.0.0.0 \
--env NEXTCLOUD_DATADIR="/nc_data" \
--env NEXTCLOUD_MEMORY_LIMIT=2048M \
--volume nextcloud_aio_mastercontainer:/mnt/docker-aio-config \
--volume /var/run/docker.sock:/var/run/docker.sock:ro \
nextcloud/all-in-one:latest

How can i edit this to include ipv6? Anyway I don't use ipv6 at all, so I don't know how this works...
Could also be something from the network, maybe a firewall, that is blocking app downloads? Is there anything that i can check if i can reach the nextcloud app store or something similiar where the apps are downloaded?

Thank you man!

**Reply by szaimen** [reply]
Do you use a vpn on the server?

**Reply by DrPleb** [reply]
Found the problem. Web app filter on customer's firewall. Thanks anyway
